datasets/mozetl/search_clients_daily/intro.md

```diff
@@ -0,0 +1,42 @@
+`search_clients_daily` is designed to enable client-level search analyses.
+Querying this dataset can be slow;
+Consider using `search_aggregates` for coarse analyses.
```

Comment:
Consider -> consider (and maybe pull it up to the previous line)

Reply:
Done